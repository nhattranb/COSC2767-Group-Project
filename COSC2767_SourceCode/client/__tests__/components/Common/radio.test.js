/*
 *   RMIT University Vietnam
 *   Course: COSC2767 Systems Deployment and Operations
 *   Semester: 2024C
 *   Assessment: Assignment 2
 *   Author: Le Dinh Ngoc Quynh
 *   ID: s3791159
 *   Created  date: 17 Dec 2024
 *   Last modified: 17 Dec 2024
 */


import React from 'react';
import { render } from '@testing-library/react';
import Radio from "../../../app/components/Common/Radio";
import {describe, expect, it, jest} from "@jest/globals";


describe ('Radio component', () => {
    it('Radio component matches snapshot', () => {
        const { asFragment } = render(<Radio handleChangeSubmit={() => {}} />);
        expect(asFragment()).toMatchSnapshot();
    });
});
