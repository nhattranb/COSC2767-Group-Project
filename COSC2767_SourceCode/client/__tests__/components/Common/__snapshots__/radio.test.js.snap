// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Radio component Radio component matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <ul>
      <li>
        <label>
          <input
            name="sorting"
            type="radio"
            value="Newest First"
          />
          Newest First
        </label>
      </li>
      <li>
        <label>
          <input
            name="sorting"
            type="radio"
            value="Price High to Low"
          />
          Price High to Low
        </label>
      </li>
      <li>
        <label>
          <input
            name="sorting"
            type="radio"
            value="Price Low to High"
          />
          Price Low to High
        </label>
      </li>
    </ul>
  </div>
</DocumentFragment>
`;
